EIPS/eip-1521.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: 1521
```

Comment:
This seems to be the wrong number. I suggest to use either 1512 (this PR) or 1523 (the discussion URL).

Reply:
I took this number because the standard "inherits" much of 721 and I like number magic

Reply:
Unfortunately that's not how number assignments work. It has to be number of the issue or pull request the EIP was posted as.

Reply:
Ok I'll use 1523